libraries/src/Application/CMSApplicationInterface.php

```diff
@@ -92,6 +92,23 @@ interface CMSApplicationInterface extends ExtensionManagerInterface, Configurati
      */
     public const MSG_DEBUG = 'debug';
 
+    /**
+    * Constant defining an enqueued message message
+    *
+    * @var    string
+    * @since  __DEPLOY_VERSION__
+    */
+    public const MSG_MESSAGE = 'message';
```

Comment:
```suggestion
    /**
     * Constant defining an enqueued message message
     *
     * @var    string
     * @since  __DEPLOY_VERSION__
     */
    public const MSG_MESSAGE = 'message';
```
Please do it like elsewhere.